### PR TITLE
[bugfix widget] Remove side pane button container when neither button enabled

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

removes container from control bar if both people and chat buttons are hidden in common controls.
# Why

image
image

# How Tested

Tested locally with widget POC app made change here too.

### CODE REVIEW

File: packages/react-composites/src/composites/common/ControlBar/CommonCallControlBar.tsx [hunk 1]
```diff
@@ -360,7 +360,7 @@ export const CommonCallControlBar = (props: CommonCallControlBarProps & Containe
             </Stack>
           </CallAdapterProvider>
         </Stack.Item>
-        {!props.mobileView && (
+        {!props.mobileView && (isEnabled(options.peopleButton) || isEnabled(options.chatButton)) && (
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/common/ControlBar/CommonCallControlBar.tsx:363]: we support injecting buttons here, what if these buttons are not enabled but a custom button is injected?

Comment (dmceachernmsft) [reply packages/react-composites/src/composites/common/ControlBar/CommonCallControlBar.tsx:363]: good point, will account for that in the logic

File: packages/react-composites/src/composites/common/ControlBar/CommonCallControlBar.tsx [hunk 2]
```diff
@@ -377,41 +377,45 @@ export const CommonCallControlBar = (props: CommonCallControlBarProps & Containe
...
-                    styles={commonButtonStyles}
-                  />
-                )}
-                {isEnabled(options?.chatButton) && chatButton}
-              </Stack>
-            </div>
-          </Stack.Item>
-        )}
+        {!props.mobileView &&
+          (isEnabled(options.peopleButton) ||
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/common/ControlBar/CommonCallControlBar.tsx:381]: consider making this a bool above to reduce the amount to parse here
```
const secondaryButtonsExist = isEnabled(options.peopleButton) || isEnabled(options.chatButton) || customButtons['secondary'] !== undefined
```

File: packages/react-composites/src/composites/common/ControlBar/CommonCallControlBar.tsx [hunk 3]
```diff
@@ -377,41 +377,45 @@ export const CommonCallControlBar = (props: CommonCallControlBarProps & Containe
...
+            isEnabled(options.chatButton) ||
+            /* @conditional-compile-remove(control-bar-button-injection) */ customButtons['secondary'] !==
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/common/ControlBar/CommonCallControlBar.tsx:383]: remove conditional compile, callwithchat uses this

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**